src/braket/aws/aws_quantum_task.py

```diff
@@ -143,6 +145,10 @@ def create(
                 IR. If the IR supports inputs, the inputs will be updated with this value.
                 Default: {}.
 
+            gate_calibrations (Optional[GateCalibrations]): A `GateCalibrations` for
+                user defined gate calibration.
+                Default: None.
```

Comment:
For these user defined gate calibrations, does the customer also need to supply fidelities? It seems weird that if I'm creating a custom calibration, I'd need to supply a fidelity value for this calibration before I run the circuit

Reply:
You do not need to supply fidelities

Reply:
It seems wrong to have to pass a data structure with these fields if they are never considered.